test.js

```diff
@@ -1,51 +1,100 @@
 const net = require('net');
 const test = require('ava');
-const pify = require('pify');
 
 const { findFreePort, isPortFree } = require('.');
 
-test('should return the passed port value', async (t) => {
+const requestedPort = 3010;
+const requestedHost = '127.0.0.1';
+const defaultHost = '0.0.0.0';
+
+const wrapper = (t, run) => new Promise(resolve => run(t, resolve));
```

Comment:
This one is just to force `net.Server` callback-style handlers to be friends with async/await?

Reply:
Yes, the idea was to end test after the server has been closed – on 'close' event.

Reply:
Okay, thanks.